src/_angular/url-field/dtv-url-field.js

```diff
@@ -73,6 +93,10 @@
               scope.invalidType = "url";
             }
 
+            if (isValid) {
+              testImage();
```

Comment:
@donnapep Seems okay then. In regards to the actual message, I don't see anywhere where you're setting the `scope.invalidType` value for `testImage()`, which message is going to get shown?